release-please-config.json

```diff
@@ -1,8 +1,6 @@
 {
   "bootstrap-sha": "798fb2d5c585f6fdee10146bfe15e03409bdddfe",
```

Comment:
nit but i think you can rollback changes to this file since its just formating

Reply:
Sorry  i was pointing to the format change in the file, not the L12 that you added. You might auto format file on save with your IDE ?

PS: There is no command to modify release-please configs so you are correct to modify them by hand

Reply:
Ah sorry thanks. I will correct

Reply:
Fixed